frontend/src/app/shared/components/autocompleter/meeting-autocompleter/meeting-autocompleter-template.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-template #labelTemplate let-item>
+    {{ item.name }} - {{ item.start_date }} {{ item.start_time }} - {{ item.project }}
+</ng-template>
+
+<ng-template #optionTemplate let-item let-search="search">
+    <div class="op-autocompleter__meeting-option-grid">
```

Comment:
I'd change this to `op-meeting-autocompleter` to match BEM

```suggestion
    <div class="op-meeting-autocompleter">
```